### DISCUSSION TITLE
Mail takes a long time to get Sent

### DISCUSSION THREAD

**Discussion by RustyDev24** [postalserver/postal#2679] [Help with using Postal]
My postal setup is just the default setup on a digital ocean droplet

I have been facing this issue where the mail takes over 75 seconds just to get queued via HTTP 
On SMTP the connection itself takes 20 seconds but the mail is queued immediately but after queuing via both the services it takes an additional 3 minutes for the mail to be sent and I want it to be immediate.

I tried running more workers `postal start --scale workers=16` but that did not seem to help for the queue clearing

**Comment by willpower232** [comment]
How much memory and CPU does your droplet have? Have you confirmed with `htop` or similar that they aren't being fully utilised?

**Reply by RustyDev24** [reply]
I have 2 cores and 4 gigs of ram and yes I have confirmed that they are not being utilised completely. This is the recommended setup I believe. Also shouldn't the sending of mail be instantaneous like every second as soon as it hits the queue ?

**Reply by RustyDev24** [reply]
will adding more cron services help ? `postal start --scale cron=3` or smth ?

**Reply by willpower232** [reply]
the workers are the ones doing the work so in theory adding more of those should have reduced the problem somewhat unless you're encountering database bottlenecks like #2620 

how much email are you trying to send?
